src/common/Executor.cpp

```diff
@@ -20,8 +20,10 @@
 #include "basic/DAXPY.hpp"
 #include "basic/REDUCE3_INT.hpp"
 #include "basic/INDEXLIST_3LOOP.hpp"
+#ifndef RUN_KOKKOS
 #include "algorithm/SORT.hpp"
 #include "apps/HALOEXCHANGE_FUSED.hpp"
+#endif
```

Comment:
Do these still have to be ifdef'd out if you have runKokkosVariant?

Reply:
Hi @MrBurmark , @crtrott -- In this PR, our intention is to focus *only* on the `basic` kernel group.  That was / is the rationale for guarding kernels from other kernel groups (i.e., apps, algorithm, etc.).

Reply:
Ok, the kernel won't be run anyway unless the variant is set as defined by the kernel constructor. I'm not sure its worth adding the ifdefs if they're easy to get wrong and temporary.

Reply:
oh right -- I'll remove this guard.   Very good point.  I'm stubbing in Kokkos versions of the kernels we're missing to see if I can solve our basic issue (i.e., the non-running of Kokkos_Lambda variants).